src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -297,7 +297,7 @@ public async Task GetAsync_StreamRefused_RequestIsRetried()
             using (Http2LoopbackServer server = Http2LoopbackServer.CreateServer())
             using (HttpClient client = CreateHttpClient())
             {
-                (_, Http2LoopbackConnection connection) = await EstablishConnectionAndProcessOneRequestAsync(client, server);
+                (_, Http2LoopbackConnection connection) = await EstablishConnectionAndProcessOneRequestAsync(client, server, 1);
```

Comment:
I don't understand why this change is necessary. Same question for the identical change in the below test.

Reply:
It was a mistake due to my misunderstanding. Fixed.